base/filesystem.jl

```diff
@@ -30,10 +30,6 @@ export File,
        JL_O_CREAT,
        JL_O_EXCL,
        JL_O_TRUNC,
-       JL_O_TEMPORARY,
-       JL_O_SHORT_LIVED,
-       JL_O_SEQUENTIAL,
-       JL_O_RANDOM,
```

Comment:
When `grep`-ing for those, I don't see them ever being used in a context of the Julia runtime. Only in that header file. What to do ?